lib/SPIRV/libSPIRV/spirv_internal.hpp

```diff
@@ -104,6 +105,7 @@ enum InternalCapability {
   ICapabilityTensorFloat32RoundingINTEL = 6425,
   ICapabilityMaskedGatherScatterINTEL = 6427,
   ICapabilityJointMatrixWIInstructionsINTEL = 6435,
```

Comment:
I see 6435 token specified twice here. Is that accurate?

Thanks

Reply:
This is by intention. We have two specifications in design, new https://github.com/intel/llvm/blob/sycl/sycl/doc/design/spirv-extensions/SPV_INTEL_joint_matrix.asciidoc and legacy https://github.com/intel/llvm/blob/sycl/sycl/doc/design/spirv-extensions/SPV_INTEL_joint_matrix_legacy.asciidoc. They both introduce 6435 token. The older one is kept in spirv_internal for backward compatibility, and the newer one is introduced to support implementation of new SPV_INTEL_joint_matrix.asciidoc specification.